### DISCUSSION TITLE
Please change colors on open tasks from Green to red, and closed tasks from red to green.

### DISCUSSION THREAD

**Discussion by RoarGronmo** [community/community#4554] [Projects and Issues]
In my world, red is for issues you should work on, green issues are issues you have accomplished.

**Comment by Lucaffo** [comment]
Mmm not everytime. A **_open_** task should be green for the keyword "open". If you **_close_** an issue should be red. You should refer to the open/close keywords and not in the concept of issue itself. An issue it's not always something wrong that need to be fixed or a problem, it could be a feature request or something similar.

**Reply by nikodemus** [reply]
Green still represents "done", and red "todo".

**Reply by Lucaffo** [reply]
Yes of course, but in github green it means open and red close. This a design choice that not anyone can understand at first, issues are not todo lists. 

When an issue is done, is closed by means, so red. When an issue is created, is opened by means, so green.

**Reply by nikodemus** [reply]
I understand the design choice just fine, I just don't agree with it. 😄 I've been using GH for years, and it still trips me up on a weekly basis. "Oh my god, why is that thing red? I thought it was already dealt with!"

You know what's a color everyone knows means "inactive, nothing to be done here"? Grey.

**Reply by RoarGronmo** [reply]
Is it meant to be a traffic light ? (Green: Go for it, Red: Don't go for it, it is dealt with (thats odd...), Yellow: Wait a little bit, more info is coming...)

**Comment by runxel** [comment]
I want actually three different colors:
- Open
- Closed, but resolved (with an accepted PR, etc.)
- Closed (declined, notabug, etc.)

**Reply by Lucaffo** [reply]
That's why they add Labels. You can create an Accepted PR, notabug label, change its color and attach to an issue. 
Idk what an accepted PR is, but you can do reviews on github issues.

**Comment by RickCogley** [comment]
Alternatively use colors that don't trip up color blind people.

**Reply by RoarGronmo** [reply]
Maybe you should then have a setting for different types of color blindness, or even "moods".

**Comment by Lucaffo** [comment]
> In my world, red is for issues you should work on, green issues are issues you have accomplished.

It's a terribile world then. Issues are not todos. Red means closed. Green it means opened.

All of you that white such things really miss the context of the issues UX. And the problems are not the color.

You really can continue to use as todo and no one care. But you think that's a todo because you use it as todo. Now you want to change some behaviour to reflect your view of the issues and that's totally a wrong.

Sorry but it hurt me see the issues tool tab threated as an other one.

**Reply by runxel** [reply]
> It's a terribile world then.  Red means closed. Green it means opened.

Well, same could be said about your world, then 🤣.

Red was, is, and will be an "attention color". If I see red, I see "a problem", something that needs my attention, right now. Something that _is closed_ and the other hand for sure doesn't need my attention any longer. So Github and you both got your colors totally wrong right now.

Sorry but it hurts me seeing that so few people understand color theory.
/s  ;)

**Reply by Lucaffo** [reply]
It's not about color theory, It's about meanings. You miss the part where the color is attributed at the function and not viceversa. 

If you attribute the function a color It's not the same to attribute to color a function. It's important to distinguish this things. 

Color theory is important, but not everytime is "alert" and not everytime green is "safe'. You need the right context. 

This is not the right context.

**Reply by selenated** [reply]
color theory *is* about meanings and what they have for people though. that's why it's being brought up. certain colors have established meanings in people's minds to begin with, and changing how you use those colors can be confusing

for example, imagine if the green reply button was red. that would make it seem like a destructive or negative action, because that association already exists in many people's heads

the point here isn't what github decided to assign the colors to- we get it, red is closed and green is open. but those words aren't the full picture for issues: they're still things you should pay attention to and red is, as said before, an attention demanding color

also, re: "github issues aren't a to do list"- they often *are* exactly that. it's a list of things that need attended to. whether those are bugs, feature requests, calls for help, or otherwise- if they're open, they usually need some form of action done with/to them :/

**Reply by Lucaffo** [reply]
I almost agree with everything you said. 

Open and Closed _**are**_ the full picture of issues. There is nothing else. It's concept proof.

As you said about red, imagine have a full red board. Could be overwhelming, plus imagine to deal with only your problems and can't close issues that are not related to you (github is mostly used to organize project with multiple people), you have constantly this board of red issues. 

I think they do like this because you should close issues with their red button, and not because you're tired to see red everywhere and click a green button to "resolve" the issue. 

As you can think changing a word like "resolve" to "close", that are similar meanings, but their little distinction is the reason of a green and of a red. 

Close something should be red because of that.

**Reply by RoarGronmo** [reply]
Today, (after been away from github desktop for a while) i realized that Closed has become Purple!! That's relaxing... Then open issues should be red (or maybe orange, not to offend someone...).

![image](https://user-images.githubusercontent.com/11217296/139042952-4ebb664c-3ae1-427c-a528-8e0a0816bed7.png)

**Reply by kmk3** [reply]
Links relevant to the red -> purple changes:

* <https://github.blog/changelog/2021-10-26-updates-to-our-issue-status-icons-and-colors/>
* <https://github.com/github/roadmap/issues/289>
